src/main/java/tatracker/model/module/Module.java

```diff
@@ -13,6 +15,7 @@
     private final String identifier;
     private final String name;
     private final UniqueGroupList groups;
+    private final UniqueSessionList doneSessions;
```

Comment:
should it contain one list for sessions that have not been completed yet as well?